### PR TITLE
#20178: Added Dell and Lenovo to warranty link function

### PR DESCRIPTION
**Pull request by akemidx**
# Description

An extension of #12477 
Dell and Lenovo were added to the list of manufacturers that pop a link to the warranty checker page.

The attemped functionality to insert the serial number upon creating the link was removed, as the links are too complicated and any change to the websites themselves will end up defaulting to the main warranty page. Therefore, to reduce bloat and create a predictable experience, the links go to each manufacturer's respected checker page.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested locally

# Checklist:

- [x] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [x] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: resources/views/hardware/view.blade.php [hunk 1]
```diff
@@ -600,6 +600,14 @@
...
+                                                    @elseif ((strtolower($asset->model->manufacturer->name) == "dell") || (str_starts_with(str_replace(' ','',strtolower($asset->model->manufacturer->name)),"dellinc")))
+                                                    <a href="https://www.dell.com/support/home/en-us?app=warranty" target="_blank">
+                                                        <img src="/img/demo/manufacturers/dellicon.png" style="width:25px;height:25px;"><span class="sr-only">Dell Warranty Status Lookup</span></i>
+                                                    </a>
+                                                    @elseif ((strtolower($asset->model->manufacturer->name) == "lenovo") || (str_starts_with(str_replace(' ','',strtolower($asset->model->manufacturer->name)),"lenovoinc")))
+                                                    <a href="https://pcsupport.lenovo.com/us/en/warrantylookup#/" target="_blank">
+                                                        <img src="/img/demo/manufacturers/lenovoicon.png" style="width:25px;height:25px;"><span class="sr-only">Lenovo Warranty Status Lookup</span></i>
```
Comment (snipe) [inline resources/views/hardware/view.blade.php]: Can I get this `sr-only` text translated (including the one above that I made for Apple).

Something like `'general.warranty_lookup_mfg' => ':manufacturer Warranty Status Lookup',`

**Review by snipe — CHANGES_REQUESTED** [review]
Few tiny changes requested :)

**Comment by snipe** [comment]
Love it, thank you!

**Comment by Robert-Azelis** [comment]
Good idea, please consider add HP warranty link https://support.hp.com/us-en/check-warranty

**Comment by akemidx** [comment]
> Good idea, please consider add HP warranty link https://support.hp.com/us-en/check-warranty
> 
> > 

Sorry for the delay, I did consider HP, but are instead working on a dynamic way for any manufacturer to have this functionality. 
Instead of adding manufacturers one by one over time.

<blockquote></blockquote>

**Pull request merged**